File: 2195-time-needed-to-buy-tickets/time-needed-to-buy-tickets.cpp
class Solution {
public:
    int timeRequiredToBuy(vector<int>& tickets, int k) {
        int count=0;
        for(int i=0;i<=k;i++){
            if(tickets[i]<=tickets[k]){
                count+=tickets[i];
            }
            else{
                count+=tickets[k];
            }
        }
        for(int i=k+1;i<tickets.size();i++){
            if(tickets[i]<tickets[k]){
                count+=tickets[i];
            }
            else{
                count+=(tickets[k]-1);
            }
        }
        return(count);
    }
};

File: 2195-time-needed-to-buy-tickets/time-needed-to-buy-tickets.py
class Solution:
    def timeRequiredToBuy(self, tickets: List[int], k: int) -> int:
        count=0
        for i in range(k+1):
            if tickets[i]<=tickets[k]:
                count+=tickets[i]
            else:
                count+=tickets[k]
        for i in range(k+1,len(tickets)):
            if tickets[i]<tickets[k]:
                count+=tickets[i]
            else:
                count+=(tickets[k]-1)
        return count
            